### PR TITLE
Fix. Anti-spam. Emails array support

### PR DESCRIPTION
**Pull request by AntonV1211**
https://doboard.com/task/9751

### CODE REVIEW

File: inc/cleantalk-ajax.php [hunk 1]
```diff
@@ -667,6 +668,7 @@ function ct_ajax_hook($message_obj = null)
     if ( apbct_is_exception_arg_request() ) {
         $base_call_params['exception_action'] = 1;
         $base_call_params['sender_info']['exception_description'] = apbct_is_exception_arg_request();
+        $base_call_params['sender_info']['$sender_emails_array'] = $sender_emails_array;
```
Comment (Glomberg) [inline inc/cleantalk-ajax.php:671]: `$base_call_params['sender_info']['$sender_emails_array']` -> `$base_call_params['sender_info']['sender_emails_array']`
Typo is in the array key name `sender_emails_array`

File: inc/cleantalk-public-integrations.php [hunk 2]
```diff
@@ -830,8 +834,12 @@ function apbct_form__formidable__testSpam($errors, $_form)
         array(
             'message'         => $message,
             'sender_email'    => $sender_email,
+            'sender_emails_array' => $sender_emails_array,
```
Comment (Glomberg) [inline inc/cleantalk-public-integrations.php:837]: Please, remove this parameter `sender_emails_array`

File: inc/cleantalk-public-integrations.php [hunk 3]
```diff
@@ -1922,6 +1931,9 @@ function apbct_form__mo_subscribe_to_email_list__testSpam()
             'sender_email'    => isset($params['email']) ? $params['email'] : '',
             'sender_nickname' => isset($input_array['mo-name']) ? $input_array['mo-name'] : '',
             'post_info'       => array('comment_type' => 'subscribe_form_wordpress_mailoptin'),
+            'sender_info'     => array(
```
Comment (Glomberg) [inline inc/cleantalk-public-integrations.php:1934]: The `sender_emails_array` must be not in root `[sender_info]` block, it must be in the own array key `[sender_info][sender_emails_array]`

**Review by Glomberg — CHANGES_REQUESTED** [review]

File: inc/cleantalk-public-integrations.php (hunk 3 above)
Comment (AntonV1211) [reply inc/cleantalk-public-integrations.php:1936]: ok

File: inc/cleantalk-public-integrations.php (hunk 2 above)
Comment (AntonV1211) [reply inc/cleantalk-public-integrations.php]: ok

File: inc/cleantalk-ajax.php (hunk 1 above)
Comment (AntonV1211) [reply inc/cleantalk-ajax.php]: ok

**Review by alexandergull — APPROVED** [review]

**Review by Glomberg — APPROVED** [review]

**Pull request merged**